### DISCUSSION TITLE
Using another system (AGE-system) and I can't get the fonts working, can I do anything or is the mod incompatible?

### DISCUSSION THREAD

**Discussion by Snarfaxi** [mclemente/fvtt-module-polyglot#342] [Q&A] [answered]
As the title says, I can't get the font working. I've tried every setting, created a new world to test if the problem was a mod and looked around here to see if anyone else has had the same problem but nothing is working. Is there anything I can do, or am I fucked? :/
Everything else works fine, I did as shown and created an item for every language and it worked great. The text is scrabbled and the one who has the item gets a translation. 
The first picture is of the fonts and the others are of my setting (even though I tried every combination). 

![6](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/264fa730-86bf-4542-860b-9f9b74915203)
![1](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/a0efb848-90f5-4331-8566-ded0d1ff683c)
![2](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/af0d3191-93eb-4e3b-bcfd-81dbad7be6b4)
![3](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/ade87e3c-a8c4-4399-bea7-1f675b4f4300)
![4](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/8cdd5b09-017e-4d23-b33f-8bb5bf5ac5d1)
![5](https://github.com/mclemente/fvtt-module-polyglot/assets/155000462/b3ef5ccb-d8cb-462f-b8f4-127f6fd3b75d)

**Comment by mclemente** [comment, marked as answer]
The module works fine, the issue is the [system's style](https://github.com/vkdolea/age-system/blob/workspace/styles/age-system.css#L4215) forces every single font in the game to be that one. You need to take this issue to the system's dev.

**Reply by Snarfaxi** [reply]
Okay, thank you for the fast response! I'll try reaching out to the system's dev to see if they have a solution. Thanks again, love the mod even if it isn't compatible to the system I use :)
